extends StaticBody3D
class_name SurfaceZone

@export var current_season: Settings.Season

# Called when the node enters the scene tree for the first time.
func _ready() -> void:
	if Settings.current_season == 5:
		var material: ShaderMaterial = load("res://Shaders/SnowShader/snow.tres")
		$NavigationRegion3D/Cube_030.set_surface_override_material(1, material)
		$NavigationRegion3D/Cube_030.set_surface_override_material(2, material)
		$NavigationRegion3D/Plane_001.set_surface_override_material(0, material)
		$NavigationRegion3D/Trimming2MiddlePieceCornerInner_003.set_surface_override_material(1, material)
		$NavigationRegion3D/Trimming2MiddlePieceCornerInner_003.set_surface_override_material(2, material)
		$NavigationRegion3D/Trimming2MiddlePieceCornerInner_003.set_surface_override_material(4, material)
		$NavigationRegion3D/Trimming2MiddlePieceCornerInner_003.set_surface_override_material(5, material)
		$NavigationRegion3D/Trimming2MiddlePieceCornerInner_003.set_surface_override_material(13, material)
		$NavigationRegion3D/Trimming2MiddlePieceCornerInner_003.set_surface_override_material(14, material)
